reference/features/README.md

```diff
@@ -11,7 +11,7 @@ Understanding our reference documentation is easier with background knowledge of
 [Distinct attribute](/reference/features/distinct.md)
 [Dumps](/reference/features/dumps.md)
 [Field properties](/reference/features/field_properties.md)
-[Filtering and faceted search](/reference/features/filtering_and_faceted_search.md)
+[Filtering and faceted search](/learn/advanced/filtering_and_faceted_search.md)
```

Comment:
Could we remove this link instead?

```suggestion
```

Conveniently, Geosearch and Sorting were never even added to this page! 😂 🤦🏻